spec/rails_helper.rb

```diff
@@ -53,12 +53,12 @@
   config.include(Shoulda::Matchers::ActiveModel, type: :form)
 
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = Rails.root.join("/spec/fixtures")
+  # config.fixture_path = Rails.root.join("/spec/fixtures")
 
   # If you're not using ActiveRecord, or you'd prefer not to run each of your
   # examples within a transaction, remove the following line or assign false
   # instead of true.
-  config.use_transactional_fixtures = true
+  config.use_transactional_fixtures = false
```

Comment:
Just wondering what the motivation was for setting this to false? As far as I'm aware transactional fixtures are way faster than the use of DB cleaner after each test

Reply:
As discussed separately - I found the cause of the test failures and have pushed a commit with a fix, and reinstated config.use_transactional_fixtures = true